packages/react-native-session-replay/ios/Tests/RCTTextViewRecorderTests.swift

```diff
@@ -0,0 +1,188 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+@testable import DatadogSDKReactNativeSessionReplay
+@_spi(Internal)
+@testable import DatadogSessionReplay
+import React
+
+internal class RCTTextViewRecorderTests: XCTestCase {
```

Comment:
exploring snapshot tests for RN would be super useful

Reply:
Yes, I want to explore this in 2 weeks. 
My other plan is to do something similar to what Xavier is doing on Android and build a DatadogCore that just writes messages to memory and that you can query later for assertions, to be used in E2E tests instead of monitors.